### PR TITLE
Change listening port from default 9000 to 9001.

### PR DESCRIPTION
**Pull request by alepolidori**
NethServer/dev#5427

Avoid conflicts with previous installation of
nethserver-rh-php56-php-fpm package that listen on 9000.

### CODE REVIEW

File: root/etc/e-smith/templates/etc/opt/rh/rh-php71/php-fpm.d/z_nethserver.conf/10base [hunk 1]
```diff
@@ -0,0 +1,2 @@
+[www]
+listen = 127.0.0.1:9001
```
Comment (gsanchietti) [inline root/etc/e-smith/templates/etc/opt/rh/rh-php71/php-fpm.d/z_nethserver.conf/10base]: You need to set the port inside the db [prop](https://github.com/NethServer/nethserver-rh-php71-php-fpm/blob/master/root/etc/e-smith/db/configuration/defaults/rh-php71-php-fpm/TCPPorts), then read the value and apply it here.

File: README.rst [hunk 2]
```diff
@@ -3,8 +3,8 @@ nethserver-rh-php71-php-fpm
 ===========================
 
 Configure default installation of PHP 7.1 running on FPM.
-The default configuration of FPM is from upstream and can freely change 
-(no template): ``/etc/opt/rh/rh-php71/php-fpm.d/www.conf``.
+The default configuration of FPM is from upstream and has
+been customized to listen on port 9001 (template: ``/etc/opt/rh/rh-php71/php-fpm.d/z_nethserver.conf``).
```
Comment (gsanchietti) [inline README.rst:7]: This template must be expanded inside the `nethserver-rh-php71-php-fpm-update` event.

**Review by gsanchietti — CHANGES_REQUESTED** [review]

File: root/etc/e-smith/templates/etc/opt/rh/rh-php71/php-fpm.d/z_nethserver.conf/10base (hunk 1 above)
Comment (alepolidori) [reply root/etc/e-smith/templates/etc/opt/rh/rh-php71/php-fpm.d/z_nethserver.conf/10base]: done

File: README.rst (hunk 2 above)
Comment (alepolidori) [reply README.rst:7]: seems already present

**Review by gsanchietti — APPROVED** [review]

**Pull request merged**